controllers/homeController.js

```diff
@@ -76,6 +87,31 @@ var findCommonChampions = function(championsA, championsB) {
     return commonChampions
 }
 
+var compareCommonChampions = function(commonChampions) {
+    var aCount = 0;
+    var bCount = 0;
+    commonChampions.forEach(function(championTuple) {
+        var cmp = compareChampionStats(championTuple[0].stats, championTuple[1].stats);
+	if (cmp > 0) {
+	    ++aCount;
+	}
+	else if (cmp < 0) {
+	    ++bCount;
+	}
+	else {
+	    // same for this champion
+	}
+    });
+
+    return aCount - bCount;
+}
+
+var compareChampionStats = function(statsA, statsB) {
```

Comment:
turns out you can also turn this into arrow notation, i tested it. BUT i dont think we should, i generally only use it on 'monads' so im curious to see what the JS way is. Like do they use it everywhere?